src/app/(auth)/subscriber/create/page.tsx

```diff
@@ -0,0 +1,43 @@
+'use client';
+
+import SubscriberForm from '@/components/Subscriber/SubscriberForm';
+import { UserGroupIcon } from '@heroicons/react/24/outline';
+import { createSubscriber } from '@/apis/subscriber';
+import { useState } from 'react';
+import { toast } from 'react-toastify';
+import { useRouter } from 'next/navigation';
+import { extractValuesFromOptions } from '@/utils/common';
+
+const Page = () => {
+  const router = useRouter();
+  const [selectedOptions, setSelectedOptions] = useState([]);
```

Comment:
I think I had this question in the previous review also. I don't see any benefit of maintaining reactive state for selected options in this component.

The options that are selected or not should be the responsibility of the SubscriberForm.